.github/workflows/promote-to-beta.yml

```diff
@@ -46,9 +46,10 @@ jobs:
             docker manifest push nextcloud/$AIO_NAME\:beta
             
             # arm64
-            docker buildx imagetools create --tag nextcloud/$AIO_NAME\:beta-arm64 nextcloud/$AIO_NAME\:develop-arm64
+            docker manifest create nextcloud/$AIO_NAME\:beta-arm64 -a nextcloud/$AIO_NAME\:develop-arm64
+            docker manifest push nextcloud/$AIO_NAME\:beta-arm64

```

Comment:
another question: based on this change, doesnt keeping the lines that I link mean that we are converting the manifests again?
https://github.com/nextcloud-releases/all-in-one/blob/eef7898f5c733b849ffe9ed2e88016f3afda9e9a/.github/workflows/promote-to-latest.yml#L63-L64

Reply:
So wouldn't we actually need to change it there as well?

Reply:
I would say no, since buildx then has a manifest and a manifets list as base. But would the savest way not be to build the arm64 tag also multiarch, even if it is called arm64? since multiarch works

Reply:
Basically we could then get rid of docker buildx imagetoosl, right?

Reply:
> Basically we could then get rid of docker buildx imagetoosl, right?

When https://github.com/docker/cli/pull/3994 is merged, released and available, we could use manifest create also inside the latest workflow, but I think there are no advantages

> > But would the savest way not be to build the arm64 tag also multiarch, even if it is called arm64? since multiarch works
> 
> Yeah, you have a point here.
> 
> Okay, would you change the workflows to implement this? :)

done

Reply:
> When https://github.com/docker/cli/pull/3994 is merged, released and available, we could use manifest create also inside the latest workflow, but I think there are no advantages

All right!

> done

Thanks! :)